braid/braid.c

```diff
@@ -1840,3 +1896,82 @@ braid_SetTimings(braid_Core core,
    return _braid_error_flag;
 }
 
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+braid_Int
+braid_SetDeltaCorrection(braid_Core           core,      
+                         braid_Int            rank,      
+                         braid_PtFcnInitBasis init_basis,
+                         braid_PtFcnInnerProd inner_prod)
+{
+   /* TODO: compatibility checks */
+   if (_braid_CoreElt(core, richardson))
+   {
+      _braid_printf("  Braid: Delta correction is not currently compatible with Richardson error estimation, (feature coming soon?) disabling Richardson\n");
+      braid_SetRichardsonEstimation(core, 0, 0, 0);
+   }
+
+   if (_braid_CoreElt(core, adjoint))
+   {
+      _braid_printf("  Braid: Delta correction is not currently compatible with the adjoint feature, (coming soon?) not enabling Delta correction\n");
+      return _braid_error_flag;
```

Comment:
why return _braid_error_flag here, but not in other if statements?  maybe remove?